pkg/tracing/tracing.go

```diff
@@ -253,7 +253,7 @@ func (t *Tracer) StartTraceContainer(mntns uint64, pid uint32, eventType EventTy
 	}
 	var eventTypesToStart []EventType
 	if eventType == AllEventType {
-		eventTypesToStart = []EventType{NetworkEventType, DnsEventType, ExecveEventType, CapabilitiesEventType, OpenEventType}
+		eventTypesToStart = []EventType{NetworkEventType, DnsEventType, ExecveEventType, CapabilitiesEventType, OpenEventType, RandomXEventType}
```

Comment:
Move this to KubeCop, this should not start when creating application profiles since this information does not part of the application profile

Reply:
done